src/components/lyne-clock/lyne-clock.scss

```diff
@@ -0,0 +1,95 @@
+:host {
+  display: block;
+  --clock-hours-animation-start-angle: 0deg;
+  --clock-seconds-animation-start-angle: 0deg;
+  --clock-animation-play-state: paused;
+}
+
+.clock {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  contain: content;
+}
+
+.clock__hand-hours,
+.clock__hand-minutes,
+.clock__hand-seconds {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  transform-origin: center center;
+  transform: rotateZ(0deg);
+}
+
+.clock__hand-hours {
+  animation-name: rotate;
+  animation-duration: 43200s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+  animation-play-state: var(--clock-animation-play-state);
+}
+
+.clock__hand-hours.clock__hand-hours--initial-hour {
```

Comment:
I am confused about these BEM selectors. In my opinion, when using a modifier class, you should not have to use the element class as well, e.g. `.clock__hand-hours--initial-hour` instead of `.clock__hand-hours.clock__hand-hours--initial-hour`. If you're doing it because of specifity, to make sure these styles get precedence over the styles of `.clock__hand-hours`, that probably means that you should have to define kind of a default modifier for the styles you are overwriting here, like so:
```
.clock__hand-hours {
  animation-timing-function: linear;
}

.clock__hand-hours--default {
  animation-name: rotate;
  animation-iteration-count: infinite;
  animation-duration: 43200s;
  animation-play-state: var(--clock-animation-play-state);
}

.clock__hand-hours--initial-hour {
  animation-name: handHoursAnimationInitialHour;
  animation-iteration-count: 1;
  animation-duration: var(--clock-hours-animation-duration);
  animation-play-state: var(--clock-animation-play-state);
}
```

Reply:
You're absolutey right, the specifity was obsolete.